core/src/assets/defaultConfig.json

```diff
@@ -60,6 +60,28 @@
           ]
         }
       },
+      "DISABLED_NODES": {
+        "isEnabled": true,
```

Comment:
I don't think we need to add it to the default config since it does nothing here. I think it should be disabled by default.